service/template.go

```diff
@@ -66,12 +66,13 @@ func getTemplate(w http.ResponseWriter, r *http.Request, envId string) (int, err
 	catalogName, templateName, templateBase, revisionOrVersion, _ := parse.TemplateURLPath(catalogTemplateVersion)
 
 	template := model.LookupTemplate(db, envId, catalogName, templateName, templateBase)
+
 	if template == nil {
 		return http.StatusNotFound, errors.New("Template not found")
 	}
 
 	if revisionOrVersion == "" {
-		if r.URL.RawQuery != "" && strings.EqualFold("image", r.URL.RawQuery) {
+		if _, ok := r.URL.Query()["image"]; r.URL.RawQuery != "" && ok {
```

Comment:
Whoops, just noticed: no need for the r.URL.RawQuery != "" check, since your `ok` check is enough on its own